src/licensedcode/data/rules/gpl-3.0-plus_22.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - gpl-3.0-plus

```

Comment:
@rakeshbalusa Things seem a bit more complicated:
from #180 the source RPM is there:
http://vault.centos.org/6.7/os/Source/SPackages/automake-1.11.1-4.el6.src.rpm
 The top level COPYING is GPL 2.0
The COPYING found side by side with ansiknr is a GPL 3.0 alright but this is really for automake.
ansiknr predates the GPL 3.0 by a long while. It would rarely be GPL-3.0 licensed if ever. 
So even though if may sound confusing to you, it is best to report this as GPL 2.0, or actually since strictly no version is stated in the file it could be technically any version.

So for now, use GPL 2.0 and add a note to the rule stating that no version is specified and that the original archive has both a GPL 2.0 and 3.0 texts. ... basically provide the explanations told here.